### ISSUE TITLE
SyntaxWarning when installing plinth

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/Plinth#336]
When installing a plinth package, I get the following warning:
```
/usr/lib/python3/dist-packages/plinth/modules/diagnostics/diagnostics.py:88: SyntaxWarning: name '_running_task' is used prior to global declaration
  global _running_task
```
I don't think this causes any noticeable issue, but it may cause unnecessary alarm for those installing the package.

**Comment by jvalleroy** [comment]
Fixed in #452.
